domains/generic.json

```
@@ -0,0 +1,13 @@
+{
+        "owner": {
+           "username": "xgeneric",
+           "email": "",
+           "discord": "1194494419064328296",
+           "OWL": "eyJlbmMiOiJBMTI4Q0JDLUhTMjU2IiwiYWxnIjoiUlNBLU9BRVAiLCJraWQiOiJaa1VsRmRqVThiUEstLXVVM2JJR09PVHFYYVFFS1ZINFVXOW53MTR6WTJnIn0.jL6LDyd2SQOHn6_qtQbQolYYL-9xQ0-jjEzNzMiteQUsdaa0HX8dVvkM21JcCuxEZbJPokrRdFQRAeeBrF45XMMg4NynXkwzJewMbGkY-oahqIcdm4LrX0h-gkbl-1zlnZJQP94JkjxBa93rnyMgGaX4DkCJpzpdrODlpxPDoTDPn7Mhg8rqU2nKhXoC1WD2fWTdQAAWrl5cBAT1VR0aAPISL5pzOrLMB3ldVxHCGcGBBEzsqW2EJRGY6UjAwQ0FE7IH-4wyewg4M6yQtjT-Cntz6Uxg5iBxA_bkCnsSjhQuE0PtfBkuIz-Pv2TF6QYHLUJdzJKs0lgYgeZGV4cHuQ.JplpmnheJVC0WyaVM3X4YA.udfWABodxYzHr_mFAj82UYRZqaySGLV-_EzZVuwkzG3X_J-GmEgGqkU6C6G6JabpJWpaO7pFzDTaPx49GirTzdGKCziFQ9pvFbOYYBx-QZxg8oZILvQY2886y8C2Pcr_.0Wh04zFffxpmI3dikZKF8Q"
+        },
+    
+        "record": {
+            "CNAME": "https://pandi.paulswebhelp.com/"
```

```suggestion
            "CNAME": "pandi.paulswebhelp.com"
```
CNAME records cannot contain protocols (`https://`) or slashes (`/`).